build/resources/main/data/items.json

```diff
@@ -0,0 +1,38 @@
+[{
+  "type": "Item",
+  "name": "apple",
+  "description": "A delicious looking apple!",
+  "consumable": true,
+  "active": true,
+
+  "stat": {
+    "name": "health",
+    "value": 10
+  }
+},
+
+  {
+    "type": "Item",
+    "name": "key",
+    "description": "It looks like a key that unlocks something.",
```

Comment:
A key that unlocks something is just like saying 'a key'. spice it up some more!